packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -369,14 +359,6 @@ export class FilesService {
     ): Promise<FileInterface> {
         let result: FileInterface | undefined = undefined;
         try {
-            if (folderId) {
-                const folder = await this.foldersService.findOneById(folderId);
-                if (!folder) throw new Error(`Folder ${folderId} not found`);
-                if (!this.contentScopeService.scopesAreEqual(folder.scope, scope)) {
-                    throw new Error("Folder scope doesn't match passed scope");
-                }
-            }
```

Comment:
I moved the authorization logic from the service to the controller. As far as I remember, we once decided to do handle authorization in the controllers and resolvers instead of the services, so this should be fine.